templates/courant/mail.html.twig

```diff
@@ -1,147 +1,476 @@
+                      <td class="inhoudKolom">
+                        <div class="inhoudKop">
+                          <h4><u>{{ catName }}</u></h4>
+                        </div>
+                        {% for bericht in berichten %}
+                          {% if bericht.cat.value == categorie %}
+                            <li class="liLink">
+                              <a href="#{{ bericht.id }}" class="link">
+                                {{ bericht.titel | bbcode("mail") }}
+                              </a>
+                            </li>
+                          {% endif %}
+                        {% endfor %}
+                      </td>
+                    {% endif %}
+                  {% endfor %}
+                  </tr>
+                  </table>
+
+                  <table role="presentation" border="0" cellpadding="0" cellspacing="0">
+                    <tr > 
+                      <td class="medium-donkerLiBo">
+                        <div class="backslash">
+                          <hr>
+                        </div>
+                      </td>
+                    </tr>
+                  </table>                  
+
+                  <!-- 
+
+                  //Berichten generator// 
+
+                  middelblauw -> lichtblauw  /
+                  lichtblauw  -> donkerblauw \
+                  donkerblauw -> lichtblauw  /
+                  lichtblauw  -> middelblauw \
+
+                  donker: #11273a
+                  medium: #043763
+                  licht:  #4e698b
+                  -->
+
+                  {% for bericht in berichten %}
+                    {% if loop.index % 2 == 0 %}
+                      {% set slashmargin = 'slashLiOn' %}
+                      {% set slashbgdirection = 'forwardslash' %}
+                    {% else %}
+                      {% set slashmargin = 'slashLiBo' %}
+                      {% set slashbgdirection = 'backslash' %}
+                    {% endif %}
+
+                    {% if loop.index % 4 == 0 %}
+                      {% set bgcolor = '#043763' %}
+                      {% set slashbgcolor = 'medium-lichtLiOn' %}
+                    {% elif loop.index % 4 == 1 %}
+                      {% set bgcolor = '#4e698b' %}
+                      {% set slashbgcolor = 'licht-donkerLiBo' %}
+                    {% elif loop.index % 4 == 2 %}
+                      {% set bgcolor = '#11273a' %}
+                      {% set slashbgcolor = 'donker-lichtLiOn' %}
+                    {% else %}
+                      {% set bgcolor = '#4e698b' %}
+                      {% set slashbgcolor = 'licht-mediumLiBo' %}
+                    {% endif %}
+
+                    <table role="presentation" border="0" cellpadding="0" cellspacing="0" style="background-color: {{bgcolor}}">
+                      <td class="{{slashmargin}}">
+                        <h3><u><a id="{{ bericht.id }}"></a>{{ bericht.titel | bbcode("mail") }}</u></h3>
```

Comment:
`<em>` ipv `<u>`, of miss geen underline als er geen link is. Je kan sowieso ieder element een id geven en er met `#id` naar verwijzen, hoeft geen `<a>` tag te zijn.